quodlibet/po/fr.po

```diff
@@ -5661,156 +5628,147 @@ msgid "Display remaining time"
+msgstr "Supprimer les titres sélectionnés"
 
 #: ../quodlibet/qltk/shortcuts.py:30
 msgid "Show the inline search entry"
-msgstr ""
+msgstr "Afficher le contenu de la recherche interne"
 
 #: ../quodlibet/qltk/shortcuts.py:31
 msgid "Left click on a column header"
-msgstr ""
+msgstr "Clic gauche sur le titre de la colonne"
 
 #: ../quodlibet/qltk/shortcuts.py:32
 msgid "Add the column to the list of columns to sort by"
-msgstr ""
+msgstr "Ajouter la colonne à la liste des colonnes à trier"
 
 #: ../quodlibet/qltk/shortcuts.py:34
 msgid "Tree View"
-msgstr ""
+msgstr "Vue arborescente"
 
 #: ../quodlibet/qltk/shortcuts.py:36 ../quodlibet/qltk/shortcuts.py:41
 msgid "Collapses the element or select the parent element"
-msgstr ""
+msgstr "Replier l'élément ou sélectionner l'élément parent"
 
 #: ../quodlibet/qltk/shortcuts.py:37
 msgid "Expands the element"
-msgstr ""
+msgstr "Déplier l'élément"
 
 #: ../quodlibet/qltk/shortcuts.py:39
 msgid "Text Entries"
-msgstr ""
+msgstr "Entrées texte"
 
 #: ../quodlibet/qltk/shortcuts.py:42
 msgid "Redo the last undone change"
-msgstr ""
+msgstr "Rétablir les derniers changement annulés"
 
 #: ../quodlibet/qltk/shortcuts.py:45
 msgid "Select all songs in all panes"
-msgstr ""
+msgstr "Sélectionner tous les titres des panneaux"
 
 #: ../quodlibet/qltk/songlist.py:377
 #, python-format
 msgid "_Filter on %s"
 msgstr "_Filtrer sur %s"
 
-#: ../quodlibet/qltk/songlist.py:1091
-#, fuzzy
+#: ../quodlibet/qltk/songlist.py:1094
 msgid "All _Headers"
-msgstr "Colonnes « _Album »"
+msgstr "Toutes les entêtes"
 
-#: ../quodlibet/qltk/songlist.py:1092
+#: ../quodlibet/qltk/songlist.py:1095
 msgid "_Track Headers"
 msgstr "Colonnes « _Piste »"
 
-#: ../quodlibet/qltk/songlist.py:1093
+#: ../quodlibet/qltk/songlist.py:1096
 msgid "_Album Headers"
 msgstr "Colonnes « _Album »"
 
-#: ../quodlibet/qltk/songlist.py:1094
+#: ../quodlibet/qltk/songlist.py:1097
 msgid "_People Headers"
 msgstr "Colonnes « Ar_tistes »"
 
-#: ../quodlibet/qltk/songlist.py:1095
+#: ../quodlibet/qltk/songlist.py:1098
 msgid "_Date Headers"
 msgstr "Colonnes « _Date »"
 
-#: ../quodlibet/qltk/songlist.py:1096
+#: ../quodlibet/qltk/songlist.py:1099
 msgid "_File Headers"
 msgstr "Colonnes « _Fichier »"
 
-#: ../quodlibet/qltk/songlist.py:1097
+#: ../quodlibet/qltk/songlist.py:1100
 msgid "_Production Headers"
 msgstr "Colonnes « _Production »"
 
-#: ../quodlibet/qltk/songlist.py:1112
-#, fuzzy
+#: ../quodlibet/qltk/songlist.py:1115
 msgid "_Customize Headers…"
-msgstr "_Configurer les colonnes..."
+msgstr "Colonnes « _Personnalisées »"
 
-#: ../quodlibet/qltk/songlist.py:1117
+#: ../quodlibet/qltk/songlist.py:1120
 msgid "_Expand Column"
-msgstr ""
+msgstr "Déplier la colonne"
 
 #: ../quodlibet/qltk/songsmenu.py:36
 #, python-format
 msgid "Run the plugin \"%(name)s\" on %(count)d song?"
 msgid_plural "Run the plugin \"%(name)s\" on %(count)d songs?"
-msgstr[0] ""
-msgstr[1] ""
+msgstr[0] "Lancer le greffon \"%(name)s\" sur la piste %(count)s ?"
```

Comment:
Same, count is the number of tracks
-> "`Lancer le greffon \"%(name)s\" sur %(count)s titre`"